test/unit/base_test.rb

```diff
@@ -50,6 +50,14 @@ class Aggregate::BaseTest < ActiveSupport::TestCase
         expected_json_string = ActiveSupport::JSON.encode(@decoded_aggregate_store)
         assert_equal expected_json_string, @instance.to_json
       end
+
+      should "generate a string with all the attributes after just being built from json" do
```

Comment:
Could we test the specific case of serializing an aggregate that was loaded from json and is in an array:

[AggregateSample.from_json(...)].to_json